src/schema/v2/conversation/index.ts

```diff
@@ -270,9 +271,28 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    fromProfile: {
+      description:
+        "The collector profile of the user who initiated the conversation",
+      type: CollectorProfileType,
+      resolve: async ({ from_id }, _options, { collectorProfilesLoader }) => {
+        if (!collectorProfilesLoader)
+          throw new Error(
+            "A X-Access-Token header is required to perform this action."
+          )
+
+        const { body: profiles } = await collectorProfilesLoader({
+          user_id: from_id,
+        })
+
+        return profiles[0]
```

Comment:
Do we have all the time an element inside the profiles array?

Reply:
If for some reason no collector profile exists, I believe this will resolve to `null` (which is the best we can do, right?).

Reply:
yes, the only problem is if `profiles` returns null, then we might see some `Uncaught ReferenceError`